flake.nix

```diff
@@ -2,30 +2,59 @@
   description = "RosControl linear feedback controller with pal base estimator and RosTopics external interface.";
 
   inputs = {
-    nix-ros-overlay.url = "github:lopsided98/nix-ros-overlay/master";
-    linear-feedback-controller-msgs = {
-      url = "github:loco-3d/linear-feedback-controller-msgs/humble-devel";
-      inputs.nix-ros-overlay.follows = "nix-ros-overlay";
-    };
+    gepetto.url = "github:gepetto/nix";
+    flake-parts.follows = "gepetto/flake-parts";
+    nixpkgs.follows = "gepetto/nixpkgs";
+    nix-ros-overlay.follows = "gepetto/nix-ros-overlay";
+    treefmt-nix.follows = "gepetto/treefmt-nix";
   };
 
   outputs =
-    { linear-feedback-controller-msgs, nix-ros-overlay, self, ... }:
-    nix-ros-overlay.inputs.flake-utils.lib.eachDefaultSystem (
-      system:
-      let
-        pkgs = import nix-ros-overlay.inputs.nixpkgs {
-          inherit system;
-          overlays = [ nix-ros-overlay.overlays.default ];
-        };
-      in
-      {
-        packages = {
-          default = self.packages.${system}.linear-feedback-controller;
-          linear-feedback-controller = pkgs.callPackage ./default.nix {
-            inherit (linear-feedback-controller-msgs.packages.${system}) linear-feedback-controller-msgs;
+    inputs:
+    inputs.flake-parts.lib.mkFlake { inherit inputs; } {
+      systems = [ "x86_64-linux" ];
+      imports = [ inputs.treefmt-nix.flakeModule ];
+      perSystem =
+        {
+          lib,
+          pkgs,
+          system,
+          self',
+          ...
+        }:
+        {
+          _module.args.pkgs = import inputs.nixpkgs {
+            inherit system;
+            overlays = [
+              inputs.nix-ros-overlay.overlays.default
+              inputs.gepetto.overlays.default
+            ];
+          };
+          checks = lib.mapAttrs' (n: lib.nameValuePair "package-${n}") self'.packages;
+          packages = {
+            default = self'.packages.linear-feedback-controller;
+            linear-feedback-controller = pkgs.rosPackages.humble.linear-feedback-controller.overrideAttrs {
```

Comment:
Should we do something like:
```nix
            humble-linear-feedback-controller = pkgs.rosPackages.humble.linear-feedback-controller.overrideAttrs {
            ...
            }
            jazzy-linear-feedback-controller = pkgs.rosPackages.jazzy.linear-feedback-controller.overrideAttrs {
            ...
            }
```

Reply:
this repository separate humble and jazzy via git branches. I think this should be followed. If we decide to separate jazzy and humble in another way, yes, we could to this.